packages/dds/tree/src/shared-tree/treeApiAlpha.ts

```diff
@@ -51,12 +53,20 @@ import {
 	type FieldBatchEncodingContext,
 } from "../feature-libraries/index.js";
 import { independentInitializedView, type ViewContent } from "./independentView.js";
+import { SchematizingSimpleTreeView, ViewSlot } from "./schematizingTreeView.js";
 
 /**
  * Extensions to {@link Tree} and {@link TreeBeta} which are not yet stable.
  * @sealed @alpha
  */
 export const TreeAlpha: {
+	/**
+	 * Retrieve the {@link TreeContext | context}, if any, for the given node.
+	 * @param node - The node for which to get a context.
+	 * @remarks Returns `undefined` for nodes not yet inserted into the tree - nodes are not attached to a context until they are inserted.
```

Comment:
Is this the *only* scenario in which undefined will be returned? That's my assumption, but not necessarily explicit in the comment.

Reply:
Yes it's the only scenario that we expect. I made it more explicit, thanks.